src/powershell/install.sh

```diff
@@ -149,6 +151,32 @@ if [ "${use_github}" = "true" ]; then
     install_using_github
 fi
 
+# If PowerShell modules are requested, loop through and install 
+if [ ${#POWERSHELL_MODULES[@]} -gt 0 ]; then
+    echo "Installing PowerShell Modules: ${POWERSHELL_MODULES}"
+    modules=(`echo ${POWERSHELL_MODULES} | tr ',' ' '`)
+    for i in "${modules[@]}"
+    do
+        echo "Installing ${i}"
+        pwsh -Command "Install-Module -Name ${i} -AllowClobber -Force -Scope AllUsers"|| continue
+    done
+fi
+
+if [ "${POWERSHELL_INSTALLBICEP}" = "true" ]; then
```

Comment:
Don't we need to install az powershell to make use of az bicep?

Reply:
We are installing standalone bicep, not az. My use case is actually because I want a mechanism to install a bicep file without az cli as it takes 90seconds and is a 1Gb in size. Using Powershell with bicep standalone and "Az.Resource" module is actually much smaller and much quicker than az cli 🤷‍♂️

Reply:
https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/install#azure-powershell

Reply:
No longer installing bicep